inst/testdata/synthetic_list.yml

```diff
@@ -1,26 +1,72 @@
-# List of synthethic datasets
-
-- name: finalSE_combo_1dose_many_drugs
-  file: finalSE_combo_1dose_many_drugs.RDS
-  description: 
-- name: finalSE_combo_2dose_nonoise
-  file: finalSE_combo_2dose_nonoise.RDS
-  description: 
-- name: finalSE_many_lines
-  file: finalSE_many_lines.RDS
-  description: 
-- name: finalSE_medium
-  file: finalSE_medium.RDS
-  description: test desc
-- name: finalSE_small_no_noise
-  file: finalSE_small_no_noise.RDS
-  description:
-- name: finalSE_small
-  file: finalSE_small.RDS
-  description:
-- name: finalSE_invalid_small
-  file: finalSE_invalid_small.RDS
-  description:
-- name: finalSE_combo_with_metrics
-  file: finalSE_combo_with_metrics.RDS
-  description:
+combo_2dose_nonoise:
+  name: finalMAE_combo_2dose_nonoise
+  file: finalMAE_combo_2dose_nonoise.qs
+  script: generateComboNoNoiseData
+  imported_as_example: TRUE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug is only as DrugName_2"
+combo_2dose_nonoise2:
+  name: finalMAE_combo_2dose_nonoise2
+  file: finalMAE_combo_2dose_nonoise2.qs
+  script: generateComboNoNoiseData2
+  imported_as_example: FALSE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug is also as single agent as DrugName"
+combo_2dose_nonoise3:
+  name: finalMAE_combo_2dose_nonoise3
+  file: finalMAE_combo_2dose_nonoise3.qs
+  script: generateComboNoNoiseData3
+  imported_as_example: FALSE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug does NOT have single agent response"
```

Comment:
"co-treatment drug does NOT have single agent response" How does it differ from "cell lines; co-treatment drug is only as DrugName_2"?

Reply:
Source of description https://github.com/gdrplatform/gDRtestData/blob/master/inst/scripts/generate_example_data.R#L17-L18

@bczech  How to improve that?